# This message holds operational mode for given joints
#
# names - joints names
# modes - joints modes
#
# Arrays should have same size

string[] names
TypeJointMode[] modes